.env.dev

```diff
@@ -30,3 +30,6 @@ REACT_APP_CAMPAIGN_BASE_URL=https://campaigns.dev.kyberengineering.io
 
 REACT_APP_TRANSAK_URL=https://staging-global.transak.com
 REACT_APP_TRANSAK_API_KEY=327b8b63-626b-4376-baf2-70a304c48488
+
+REACT_APP_GTM_ID=
+REACT_APP_ORIGIN=
```

Comment:
what is purpose of this variable?

Reply:
Google Tag Manager, env and stg doesn't have. But I put it for be synced with prod.

Reply:
i mean `REACT_APP_ORIGIN` didnt use in any place?